docs/_rpc/ns-eth.md

```diff
@@ -8,6 +8,28 @@ Geth provides several extensions to the standard "eth" JSON-RPC namespace.
 * TOC
 {:toc}
 
+### eth_createAccessList
+
+This method creates an access list based on a given transaction on the current pending state of the blockchain.
```

Comment:
Specify the input params, or at least say that it can be called with the same parameters as eth_call

Reply:
Please see latest commit.